### ISSUE TITLE
/api/v1/auth broke with django-oauth-toolkit 0.9.0

### ISSUE THREAD

**Issue by duvholt** [dotkom/onlineweb4#1475]
https://github.com/evonove/django-oauth-toolkit/compare/0.8.1...0.9.0

**Comment by sklirg** [comment]
Actually isn't resolved.

**Comment by sklirg** [comment]
Example error from updated version:
```
2017-03-24 22:53:38,849 ERROR django.request:284 Internal Server Error: /api/v1/transactions/
Traceback (most recent call last):
  File "/srv/www/onlineweb4/env_py3/lib/python3.4/site-packages/django/core/handlers/base.py", line 235, in get_response
    response = middleware_method(request, response)
  File "/srv/www/onlineweb4/env_py3/lib/python3.4/site-packages/reversion/middleware.py", line 34, in process_response
    self._close_revision(request, False)
  File "/srv/www/onlineweb4/env_py3/lib/python3.4/site-packages/reversion/middleware.py", line 30, in _close_revision
    _set_user_from_request(request)
  File "/srv/www/onlineweb4/env_py3/lib/python3.4/site-packages/reversion/views.py", line 18, in _set_user_from_request
    if hasattr(request, "user") and is_authenticated(request.user) and get_user() is None:
  File "/srv/www/onlineweb4/env_py3/lib/python3.4/site-packages/reversion/compat.py", line 16, in is_authenticated
    return user.is_authenticated()
AttributeError: 'NoneType' object has no attribute 'is_authenticated'
```

Should be fixed in #1836.
